i3ipc/i3ipc.py

```diff
@@ -944,6 +944,8 @@ class Con(object):
         Whether the container is floating or not. Possible values are
         "auto_on", "auto_off", "user_on" and "user_off"
 
+    .. attribute:: pid
+        The id of the process who owns the client window
```

Comment:
Can you mark this as sway only in the docs and maybe give a version number where this was included?